### PR TITLE
Add documentation for clusterContraints.k8sVersion in schema.md

### PR DESCRIPTION
**Pull request by vcanaa**

### CODE REVIEW

File: docs/schema.md [hunk 1]
```diff
@@ -628,6 +628,18 @@ x-google-marketplace:
           minimumNodeCount: 3
 ```
 
+You can also define restrictions for the version of the cluster. For example, if the application
+requires that the cluster runs k8s version greater or equals to 1.13.7, use:
```
Comment (eshiroma) [inline docs/schema.md]: nit: greater than or equal to

File: docs/schema.md [hunk 2]
```diff
@@ -628,6 +628,18 @@ x-google-marketplace:
...
+You can also define restrictions for the version of the cluster. For example, if the application
+requires that the cluster runs k8s version greater than or equal to 1.13.7, use:
+
+```yaml
+x-google-marketplace:
+  clusterConstraints:
+    k8sVersion: >=1.13.7
+```
+
+Any semver pattern is accepted, however it is recommended that the constraints are kept to a
```
Comment (huyhg) [inline docs/schema.md]:
```suggestion
Any SemVer expression is accepted; however, it is recommended that the constraints are kept to a 
minimum for the application to be deployable to a wider range of clusters.

You can use [this website](https://semver.npmjs.com/) for testing SemVer expressions.
```

**Pull request merged**